tests/test_field_now.py

```diff
@@ -0,0 +1,30 @@
+from copy import deepcopy
+import pytest
+
+from django.db.models.functions import Now
+
+from .models import TestDatetimeModelStandard
+
+
+@pytest.mark.django_db
+def test_dirty_fields_field_now():
+    tm = TestDatetimeModelStandard.objects.create()
+
+    assert tm.get_dirty_fields() == {}
+
+    tm.datetime_field = Now()
+
+    assert tm.get_dirty_fields() == {'datetime_field': None}
+
+
+@pytest.mark.django_db
+def test_dirty_fields_field_now2():
+    # we can't compare Now() values, so just test keys
+
+    tm = TestDatetimeModelStandard.objects.create(datetime_field=Now())
+
+    assert set(tm.get_dirty_fields().keys()) == {'datetime_field'}
```

Comment:
This is not a correct assertion for the general `django-dirtyfields` approach: if the model has been just created, none of its fields should be dirty,

Reply:
Seems that without full refactoring with __setattr__ or something like that we can't do this.
Sorry to bother, we can close this pull request.

Reply:
Are there any fix for that feature?